db/setup/schema.sql

```diff
@@ -2,10 +2,10 @@ CREATE DATABASE ft_transcendence;
 
 \c ft_transcendence;
 
-CREATE TABLE IF NOT EXISTS users(
-    ID          UUID DEFAULT gen_random_uuid()  PRIMARY KEY,
-    USERNAME    VARCHAR(20)                     NOT NULL UNIQUE,
-    EMAIL       VARCHAR(50)                     NOT NULL UNIQUE,
-    AVATAR      VARCHAR(150),
-    CREATED_AT  TIMESTAMPTZ                     NOT NULL DEFAULT now()
+CREATE TABLE IF NOT EXISTS Users(
+  "id" UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+  "username" VARCHAR(20) NOT NULL UNIQUE,
+  "email" VARCHAR(50) NOT NULL UNIQUE,
+  "avatar" VARCHAR(150),
+  "createdAt" TIMESTAMPTZ NOT NULL DEFAULT NOW()
```

Comment:
Here we quote the column names to have the same naming style between SQL and TypeScript.

The downside is that when we do raw queries to the database, we have to also quote the names, for example:

```sql
SELECT "id", "email" FROM Users WHERE "username" = 'example';
```